qdb/comm.py

```diff
@@ -77,6 +83,17 @@ def fmt_err_msg(error_type, data, serial=None):
         serial=serial,
     )
 
+def parse_variables(paydict):
+    """
+    parses the variables from the dictioinary and converts to string type
+    """
+    if isinstance(paydict,dict) and paydict:
+        strdict={}
+        for (k,v) in paydict.items():
```

Comment:
this loop could be turned into a comprehension like:
```python
return {repr(k): repr(v) for k, v in paydict.items()}
```

I also think we may want to use `repr` over `str` because that is how things normally are displayed in the repl or debugger.